pkgs/tools/typesetting/tex/texlive/build-texlive-package.nix

```diff
@@ -15,17 +15,27 @@
 , texliveBinaries
 }:
 
-{ pname
+/* Convert an attribute set extracted from tlpdb.nix (with the deps attribute
+  already processed) to a fake multi-output derivation with possible outputs
+  [ "tex" "texdoc" "texsource" "tlpkg" "out" "man" "info" ]
+*/
+
+# TODO stabilise a generic interface decoupled from the finer details of the
+# translation from texlive.tlpdb to tlpdb.nix
+lib.makeOverridable({ pname
```

Comment:
We might want to replace `lib.makeOverridable` with a variant of it that automatically unlocks the fixed hashes.
 
This doesn't have to be in this PR, any time before stabilizing (and announcing) the interface will do, but I wanted ot mention it somewhere.

On that note, we probably want individual `postUnpack`/`postFixup`-thingies for each of the containers (or possibly something that takes an attrset).

Reply:
Good points. Re fixed hashes, we have to plan it carefully: some operations are hash safe and invalidating the hash would be surprising (like changing `postFixup` or the formats).

Reply:
For clarity: I removed `makeOverridable` because the interface of `buildTeXLivePackage` is still unstable, so I am marking this as resolved. It will need to be addressed, but presumably after 23.11.